### PR TITLE
Soback 124 links on facets and documents

### PR DESCRIPTION
**Pull request by jesperlauridsen**
Links in document expanded fields and facets are now router-links, meaning you can open them in a different tab, and even see the URL.

@thomasegense for test of functionality

@jorntx for codereview

### CODE REVIEW

File: src/js/src/components/SearchFacetOptions.vue
Comment (jorntx) [inline src/js/src/components/SearchFacetOptions.vue:17]: Of course - should have caught this in my first review - sorry.

File: src/js/src/components/searchSingleItemComponents/SearchSingleItemAllData.vue
Comment (jorntx) [inline src/js/src/components/searchSingleItemComponents/SearchSingleItemAllData.vue:35]: What is _single-object-1-length_ a reference to? :)

**Review by jorntx — CHANGES_REQUESTED** [review]
Tried the links on facets and the app in general and besides the _single-object-1-length_ reference that triggers an error it works fine and code checks out good.

Good refactor 👍  The router links should have been there from the start - my bad 🙏

File: src/js/src/components/SearchFacetOptions.vue
Comment (jesperlauridsen) [reply src/js/src/components/SearchFacetOptions.vue:17]: :+1: No worries - I should have realised we wanted links instead of relying on normal buttons instead. Its okey now tho :-)

File: src/js/src/components/searchSingleItemComponents/SearchSingleItemAllData.vue
Comment (jesperlauridsen) [reply src/js/src/components/searchSingleItemComponents/SearchSingleItemAllData.vue]: Fixed, that was weird!

**Review by jorntx — APPROVED** [review]
That did the trick - all good from me 👍 

Awaiting approval from Egense before merging.

**Review by thomasegense — APPROVED** [review]
OK

**Pull request merged**
